snowflake-cortex/app/setup_script.sql

```diff
@@ -0,0 +1,27 @@
+-- This is the setup script that runs while installing a Snowflake Native App in a consumer account.
+-- For more information on how to create setup file, visit https://docs.snowflake.com/en/developer-guide/native-apps/creating-setup-script
+
+-- A general guideline to building this script looks like:
+-- 1. Create application roles
+CREATE APPLICATION ROLE IF NOT EXISTS app_public;
+
+-- 2. Create a versioned schema to hold those UDFs/Stored Procedures
+CREATE OR ALTER VERSIONED SCHEMA core;
+GRANT USAGE ON SCHEMA core TO APPLICATION ROLE app_public;
+-- 3. Create a view referencing the shared view in the package shared-content.sql.
+CREATE VIEW IF NOT EXISTS core.shared_view
+AS SELECT * FROM package_shared.PROVIDER_SPOTIFY_VIEW;
+
+-- 4. Grant appropriate privileges over the view to your application roles. 
+GRANT SELECT ON VIEW core.shared_view TO APPLICATION ROLE app_public;
```

Comment:
Because this is just used in the streamlit, we don't technically need to create a view and share it with the app role. We can query the package view directly, if desired.

Reply:
got it, changing it for next commit